### ISSUE TITLE
Bot Framework Emulator - Audio card -image is not displaying.

### ISSUE THREAD

**Issue by rvinothrajendran** [microsoft/BotFramework-WebChat#3315]
## Version

4.9.0

## Describe the bug

Audio card -image is not displaying.

## Steps to reproduce

Try the Audio Card sample with image.

## Expected behavior

Image should be display

## Screenshots

![image](https://user-images.githubusercontent.com/16264167/87865815-413d4300-c997-11ea-9919-417894eec4d1.png)

**Comment by v-kydela** [comment]
Thank you for the report, @rvinothrajendran 

Would you kindly link to the sample you're referring to, and provide some code that reproduces the issue?

**Comment by rvinothrajendran** [comment]
@v-kydela Kindly find the sample here : https://github.com/rvinothrajendran/BotTutorialSample/tree/master/Python_tutorial/18-AudioCard/audiocard

**Comment by v-kydela** [comment]
@compulim - I have confirmed that audio card images do not render in Web Chat or Emulator, and it's not just because of the SVG. JPG images also do not render. (This has nothing to do with the bot being written in Python, of course.) I will transfer this to you.

**Comment by daveta** [comment]
@compulim : Any updates?

**Comment by peterinnesmsft** [comment]
@compulim are you able to take a look into this one?

**Comment by compulim** [comment]
I believe this is a feature we missed.

**Comment by tdurnford** [comment]
Related to https://github.com/microsoft/BotFramework-DirectLineJS/issues/180

**Comment by tonyanziano** [comment]
So to clarify, @tdurnford, Web Chat currently supports this with Bots connected via DirectLine?

If that's the case, there's basically a lack of parity here between Emulator and Web Chat and the ask would be to implement images with audio cards in the Emulator?

**Comment by tdurnford** [comment]
@tonyanziano Not quite. If Web Chat receives an Audio Card with an image, it will render the poster correctly. However, if the bot sends an Audio Card with an image and it is sent through DirectLine, DirectLine will remove the image. Web Chat/DirectLineJs and the Emulator current exhibit the same behavior.

### Additional Context
https://github.com/microsoft/BotFramework-WebChat/blob/1559ab52158e1b3d0e523d1c2b8b66c462e0494c/packages/bundle/src/adaptiveCards/Attachment/AudioCardContent.js#L21

**Comment by corinagum** [comment]
I believe this is blocked by https://github.com/microsoft/BotFramework-DirectLineJS/issues/180, confirm?
Mark DLJS issue as bug? 
Assigning to @Kaiqb for visibility

**Comment by compulim** [comment]
@corinagum this is also related to https://github.com/microsoft/BotFramework-DirectLineJS/issues/255.

**Comment by carlosscastro** [comment]
@compulim I see that https://github.com/microsoft/BotFramework-DirectLineJS/issues/255 is scheduled to ship on R13. ASsuming that is the case, would this be also addressed in R13?
